test-suite/src/test/scala/org/scalajs/testsuite/javalib/LinkedListTest.scala

```diff
@@ -0,0 +1,357 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib
+
+import scala.language.implicitConversions
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.scalajs.runtime.UndefinedBehaviorError
+
+import java.util.LinkedList
+import java.util.Comparator
+
+object LinkedListTest extends JasmineTest {
+
```

Comment:
Given that there are *already* tests for all lists in [ListTest](https://github.com/scala-js/scala-js/blob/master/test-suite/src/test/scala/org/scalajs/testsuite/javalib/ListTest.scala), I quite agree that you should refer to those. There's no reason to delay this, as it's only a matter of a couple of lines of code to reuse all these existing tests.

Further abstraction in `AbstractSequentialListTest` takes more work, but is less critical.